opm/parser/eclipse/EclipseState/IOConfig/RestartConfig.cpp

```diff
@@ -50,10 +51,97 @@ namespace Opm {
                 && std::all_of( x.begin() + 1, x.end(), is_digit );
         }
 
+
+        std::set<std::string> notSupported = {"NORST", "SFREQ" , "STREAM"};
```

Comment:
Just a general observation here: I don't think this needs to be a set. It's three elements, and it doesn't change at all. We can make it a constexpr array and just use `any_of`, meaning the algorithm for unsupported later becomes:

```
bool unsupported( const std::string& kw ) {
    static constexpr const char* keywords[] = { "NORST", "SFREQ", "STREAM" };   
return std::any_of( std::begin( keywords ), std::end( keywords ), std::equal< std::string >() );
}
[...]

if( !unsupported( keyword ) ) insertKeyword( keyword );
```

Obviously, making it an array or something doesn't change the real benefit here which is providing a function that determines if a keyword is unsupported or not, that can also be accomplished with a string. It does lift the burden of comparison from the insert algorithm.